core/storage/user/gae_models.py

```diff
@@ -2621,9 +2622,7 @@ class PendingDeletionRequestModel(base_models.BaseModel):
     # known on the Oppia site.
     normalized_long_term_username = (
         datastore_services.StringProperty(indexed=True))
-    # Role of the user. Needed to decide which storage models have to be deleted
-    # for it.
-    role = datastore_services.StringProperty(required=True, indexed=True)
```

Comment:
I'm not sure you can just delete this, you have to deprecate it first and then delete it later I believe (because there are still models in the datastore with it). See what we did with skill_tags last time (I think @vojtechjelinek and @rohitkatlaa know the process).

Reply:
We delete this model once the deletion request is completed and we also have a sanity check for the same. In any case if there  remains one-two models which are not deleted but expected to be delete won't create any issue.

@vojtechjelinek Can you confirm whether it's fine to remove this field directly?